src/PKSim.Infrastructure/Serialization/Xml/Serializers/SimulationXmlSerializer.cs

```diff
@@ -17,6 +17,7 @@ public override void PerformMapping()
          Map(x => x.ReactionDiagramModel);
          Map(x => x.Reactions);
          Map(x => x.SimulationSettings);
+         Map(x => x.OutputMappings);
```

Comment:
I think you could add a test to check serialization. This will probably fail. But at least we remember that we have to do sthg:)

Reply:
I added the unit test, but yes indeed it fails. Let's discuss this a bit today.